### PR TITLE
Flat object before passing to tableToMardown

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

**Comment by meirwah** [comment]
@maiorrulz did you validated this fix? look like something that can break many integrations

File: Scripts/script-ToTable.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ commonfields:
 name: ToTable
 script: |-
   var headers = args.columns ? args.columns.split(',') : null;
-  return {Type: entryTypes.note, Contents: tableToMarkdown(args.title, args.data, headers), ContentsFormat: formats.markdown};
+  var flattenObject = treeToFlattenObject(args.data);
```
Comment (slavikm) [inline Scripts/script-ToTable.yml]: `ToTable` receives an array and needs to flatten the elements while keeping an array to pass to `tableToMarkdown`

**Comment by slavikm** [comment]
What is the difference between: `treeToFlattenObject` and `flattenFields`?

**Comment by maiorrulz** [comment]
I think they doing the same almost the same, except of treeToFlattenObject can flat array of objects, and if it is array of primitives, function returns array which can be printed in table

File: Scripts/script-ToTable.yml (hunk 1 above)
Comment (maiorrulz) [inline Scripts/script-ToTable.yml]: What is use case of toTable, I never used it, it is better to understand when you are making fix :-)

Comment (slavikm) [inline Scripts/script-ToTable.yml]: I added it recently to allow taking an array in the context and printing it nicely. It's basically a wrapper script for `tableToMarkdown`.

**Comment by meirwah** [comment]
@maiorrulz can we verify this PR?

**Comment by aviadl** [comment]
@meirwah @maiorrulz any reason that this is not merged ?

**Comment by meirwah** [comment]
@aviadl they were concerns this may break some integrations, asked @maiorrulz to double check to be sure

**Comment by maiorrulz** [comment]
I didn't find any integration in our content which uses **toTable** function.

**Pull request merged**